package.json

```diff
@@ -35,5 +35,6 @@
     "prettier": "~1.5.3",
     "rollup": "^0.45.2",
     "rollup-plugin-uglify": "^2.0.1"
-  }
+  },
+  "typings": "picodom.d.ts"
```

Comment:
Should this be types ([doc](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html))?